CMakeLists.txt

```diff
@@ -109,16 +109,26 @@ else()
 
   # No config file for this
   if(WITH_GFLAGS)
-    find_package(gflags REQUIRED)
+    option(GFLAGS_USE_TARGET_NAMESPACE "Use gflags import target with namespace." ON)
+    find_package(gflags CONFIG)
+    if(gflags_FOUND)
+      list(APPEND THIRDPARTY_LIBS ${GFLAGS_TARGET})
+    else()
+      find_package(gflags REQUIRED)
+      list(APPEND THIRDPARTY_LIBS gflags::gflags)
+      include_directories(${gflags_INCLUDE_DIR})
```

Comment:
there is no need to add this include globally.

Reply:
Agree, not sure why it was there previously.
It wasn't used anyway because the casing is wrong.